Pear/View/MeetupStatusView.swift

```diff
@@ -162,12 +162,12 @@ class MeetupStatusView: UIView {
             .underlinedFont("\(formatDate(date))")
             .normalFont(".")
 
-        if user.instagram != nil || user.facebook != nil {
-            fullText
-                .normalFont("\nYou can reach \(user.firstName) on ")
-                .socialMediaLinks(instagram: user.instagram, facebook: user.facebook)
-                .normalFont(".")
-        }
+//        if user.instagram != nil || user.facebook != nil {
```

Comment:
Comment out for now because backend took social media out of user model but I'm requesting they add it back. Will uncomment when that is done.